Solidus-Swift-SDK/EndpointsBase.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  EndpointsBase.swift
+//  Solidus-Swift-SDK
+//
+//  Created by Francisco Flores on 5/3/16.
+//  Copyright © 2016 Omar Guzman. All rights reserved.
+//
+
+import Foundation
+
+struct EndpointsBase {
+    static var products = "/api/products"
```

Comment:
@softr8 is only the base to construct the _"final"_ url request. I mean, those gonna be concatenated to the domain store, which could be _www.store.com_ or _https://store.com_. Also, this in order to have a cleaner and organised code.